packages/launcher/src/renderer/launcher/apps/AppsView.js

```diff
@@ -197,11 +197,19 @@ class AppsView extends Component<Props, State> {
 
   // RENDER
 
-  renderApp(app: AppData, own: boolean) {
+  renderApp(app: AppData, own: boolean, index: number) {
     return own ? (
-      <OwnAppItem ownApp={app} onOpenApp={this.onOpenApp} />
+      <OwnAppItem
+        key={`app-${index}`}
```

Comment:
Better not using the `index` as key, the app's local ID should be more relevant.